Cargo.toml

```diff
@@ -52,11 +52,19 @@ path = "examples/04_pong/main.rs"
 name = "assets"
 path = "examples/06_assets/main.rs"
 
+[[example]]
+name = "physics"
+path = "examples/07_physics/main.rs"
+
 [dev-dependencies]
 obj-rs = "0.4.16"
 gfx = "0.12"
 
 [dependencies]
 cgmath="0.11"
+nalgebra = "0.9.*"
```

Comment:
I've moved these back to being straight dependencies instead of dev dependencies, with the introduction of the physics processor. Let me know if I should switch them back.